src/providers/introduction.stories.mdx

```diff
@@ -119,21 +119,22 @@ import {
   MeetingProvider,
   useMeetingManager
 } from 'amazon-chime-sdk-component-library-react';
+import { MeetingSessionConfiguration } from 'amazon-chime-sdk-js';
 
 const MyApp = () => {
   const meetingManager = useMeetingManager();
 
   const joinMeeting = async () => {
     const response = await fetch('/my-meetings-endpoint');
     const data = await response.json();
-    const joinData = {
-      meetingInfo: data.Meeting,
-      attendeeInfo: data.Attendee,
-      eventReporter: new NoOpEventReporter()
-    };
+    
+    let meetingSessionConfiguration = new MeetingSessionConfiguration(data.JoinInfo.Meeting, data.JoinInfo.Attendee);
 
     try {
-      await meetingManager.join(joinData);
+      await meetingManager.join({
+        meetingSessionConfiguration,
+        eventReporter: new NoOpEventReporter()
```

Comment:
nit: trailing comma

Why we have so many similar examples, can we remove some of them....?

Reply:
Done